node/node.go

```diff
@@ -360,6 +376,7 @@ func makeNodeInfo(config cfg.Config, sw *p2p.Switch, privKey crypto.PrivKeyEd255
 			cmn.Fmt("p2p_version=%v", p2p.Version),
 			cmn.Fmt("consensus_version=%v", consensus.Version),
 			cmn.Fmt("rpc_version=%v/%v", rpc.Version, rpccore.Version),
+			cmn.Fmt("tx_indexer=%v", config.GetString("tx_indexer")),
```

Comment:
I mean, peers have no use for those field values.  What should we do when someone implements a new type of index, like "redis"?  The behavior is undefined.

If you want, we can create two fields, like "tx_index=on", "tx_index_backend="kv" or something, for diagnostic info.  I imagine that it's just exposing vulnerabilities though.